c2corg_ui/templates/waypoint/view.html

```diff
@@ -73,7 +73,7 @@
 
 <header class="view-details-title">
   <h1>${locale['title']}
-    <label class="badge waypoint" onClick="window.location='${request.route_url('waypoints_index')}' ">
+    <label class="badge waypoint" ng-click="window.location='${request.route_url('waypoints_index')}&wt=${waypoint['waypoint_type']}&bbox=' ">
```

Comment:
In addition the ``wt`` parameter does not exist. I think you want to use ``wtyp`` (see the URL parameters in the waypoints advanced search page).